packages/leafygreen-provider/src/LeafyGreenProvider.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { createContext, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useEventListener } from '@leafygreen-ui/hooks';
+
+interface UsingKeyboardState {
+  usingKeyboard?: boolean;
+  setUsingKeyboard?: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+export const UsingKeyboardContext = createContext<UsingKeyboardState>({
+  usingKeyboard: undefined,
+  setUsingKeyboard: undefined,
+});
+
+export const NavigationKeyCodes = {
+  tab: 9,
+  leftArrow: 37,
+  upArrow: 38,
+  rightArrow: 39,
+  downArrow: 40,
+} as const;
+
+type NavigationKeyCodes = typeof NavigationKeyCodes[keyof typeof NavigationKeyCodes];
```

Comment:
Are we using this type anywhere? Do we need to define it?

Reply:
We only use it in this file and the tests. Will remove the type.